sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/DiagnosticsInfo.cs

```diff
@@ -46,7 +46,7 @@ public partial class DiagnosticsInfo
         private IDictionary<string, BinaryData> _serializedAdditionalRawData;
 
         /// <summary> Initializes a new instance of <see cref="DiagnosticsInfo"/>. </summary>
-        public DiagnosticsInfo()
+        internal DiagnosticsInfo()
```

Comment:
we need to figure out a reason for changes like this.
Looks like the usage of this model has changed.

Reply:
I guess this is because of the TCGC usage fix

Reply:
We saw same changes in the test projects as well therefore i believe this is fine